extensions/wikia/ArticleVideoMVP/ArticleVideoMVP.hooks.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class ArticleVideoMVPHooks {
+	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
+		global $wgVideoMVPArticles, $wgCityId;
+
+		$wg = F::app()->wg;
+		$articleId = $wg->Title->getArticleID();
+
+		if ( isset( $wgVideoMVPArticles[$wgCityId] ) &&
+		     isset( $wgVideoMVPArticles[$wgCityId][$articleId] )
+		) {
+			\Wikia::addAssetsToOutput( 'premium_mvp_scss' );
+			\Wikia::addAssetsToOutput( 'premium_mvp_js' );
+		}
+
+		return true;
+	}
+
+	public static function onMakeGlobalVariablesScript( array &$vars, OutputPage $out ) {
+		global $wgVideoMVPArticles, $wgCityId;
+
+		$wg = F::app()->wg;
+		$ooyalaPlayerId = $wg->OoyalaApiConfig['playerId'];
+		$articleId = $vars['wgArticleId'];
+		$jsFile = 'http://player.ooyala.com/v3/' . $ooyalaPlayerId . '?platform=html5-priority';
```

Comment:
Can we make a constant for this variable?
It's already used in https://github.com/Wikia/app/blob/54b798e11a9e274f19c4f84053ca29557ea1b9e8/extensions/wikia/VideoHandlers/handlers/OoyalaVideoHandler.class.php#L20-L20